app/views/assignments/_form.html.erb

```diff
@@ -124,6 +150,16 @@
         .attr('value', JSON.stringify(updated_restrictions))
         .appendTo('#assignmentForm');
   }
+  function generateLtiKeys(){
+    let lti_credentials_needed = false;
+    if ($('#lms-integration-check').is(":checked")) {
```

Comment:
See this https://apidock.com/rails/ActionView/Helpers/FormHelper/check_box

Reply:
done